bots/pr/src/main/java/org/openjdk/skara/bots/pr/PullRequestCheckIssueVisitor.java

```diff
@@ -263,7 +263,8 @@ public void visit(SymlinkIssue issue) {
 
     @Override
     public void visit(BinaryIssue issue) {
-        log.fine("ignored: binary file");
+        addFailureMessage(issue.check(), "The binary file " + issue.path().toString() + " is not allowed in this repository.");
```

Comment:
Can you make the message match the ones for executable files and symbolic links above?

Reply:
What about `String.format("Binary files are not allowed (file: %s)", issue.path())`?